proposals/NNNN-resource-instance-analysis.md

```diff
@@ -0,0 +1,71 @@
+* Proposal: [NNNN - Resource Instance Analysis](NNNN-resource-instance-analysis.md)
+* Author(s): [Ashley Coleman](https://github.com/V-FEXrt)
+* Status: **Design In Progress**
+
+## Introduction
+
+This proposal introduces a new term, Resource Instance, which represents a 
+unique and specific resource. The instance is referenced via a handle returned
+from the create handle intrinsics and it is possible to have multiple handles
+pointing at a specific Resource Instance, but each unique instance is
+disambiguated by the handle creation parameters.
+
+Beyond creation parameters, Resource Instances have several pieces of associated
+metadata. Type, CounterDirection (HasCounter), and GloballyCoherent are
```

Comment:
Please replace "metadata" with some other term like "property", "feature" or "characteristic". The term metadata is already used in LLVM to attach extra info to instructions or global objects ([see LLVM metadata](https://llvm.org/docs/LangRef.html#metadata)). The term "attribute" is also used by both Clang and LLVM.